pkgs/by-name/en/entangled/package.nix

```
@@ -0,0 +1,53 @@
+{
+  python3Packages,
+  lib,
+  pkgs,
```

```suggestion
  brei,
  copier,
```
To make overrides easier we generally do not use pkgs as argument.